jdaviz/configs/default/plugins/export/export.py

```diff
@@ -200,11 +242,21 @@ def export(self, filename=None, show_dialog=None):
                 self.save_movie(viewer, filename, filetype)
             else:
                 self.save_figure(viewer, filename, filetype, show_dialog=show_dialog)
+
         elif len(self.table.selected):
-            filetype = self.table_format.selected
-            if not filename.endswith(filetype):
-                filename += f".{filetype}"
+
+            if "." not in filename:
+                filename += ".ecsv"

```

Comment:
Thanks to terrible astronomical naming conventions, it's possible (if regrettable) that folks would put a `.` in their filename without representing an extension, often for target names, like "2MASSJ2048+25.4000.fits". `endswith(f".{extension}")` is safer here.

Reply:
changed that, i picked up the wrong commit when rebasing but that's whats done in main